pkgs/tools/nix/nixos-render-docs/src/nixos_render_docs/manual.py

```diff
@@ -89,7 +120,7 @@ def _parse_included_blocks(self, token: Token, block_args: dict[str, str]) -> No
             try:
                 self._base_paths.append(path)
                 with open(path, 'r') as f:
-                    tokens = self._parse(f.read())
+                    tokens = self._parse(f.read(), auto_id_prefix=block_args.get("auto-id-prefix", None))
```

Comment:
Changed; We prepend the line number of the current include to prevent duplicate ids now. 

I think it could still happen that we have duplicate ids. If someone uses the same id_prefix somewhere else?  
But i think its fine to ask our users to add unique auto_id_prefixes for each include, where we use it. (Currently only 1 place anyways)

Reply:
Nah i'll rather want this merged soon. And work on that problem later.